test/integration/test-example-validation.js

```diff
@@ -54,6 +54,7 @@ describe('example', function() {
     'instagram.amp.html',
     'released.amp.html',
     'soundcloud.amp.html',
+    'springboard-player.amp.html',
```

Comment:
The right thing to do is to whitelist it below. We remove the whitelist when we fixed validation.